docs/customization/customizing-emails.mdx

```diff
@@ -0,0 +1,174 @@
+---
+sidebar_position: 6
+---
+
+# Customizing Transactional Emails
+
+This guide will teach you how to customize transactional emails provided by Solidus.
+
+Solidus has built-in transactional emails that notify customers of various events associated with their order.
+For example, the following actions can trigger an email:
+
+- Completing order checkout
+- Shipping an order
+- Cancelling an order
+- Completing a batch promtion codes creation (in the admin panel)
+
+Solidus has built-in emails for all of the above scenarios and more. However, given that the default Solidus
+emails are intended to be very plain, you will likely wish to customize them for your store.
+You may also want to add new emails, such as a welcome email when a customer creates a new account, an
+abandoned cart email when a customer leaves their cart without checking out, or a notification that a
+customer's credit cart has expired if your store uses recurring subscriptions.
+
+:::info
+
+Solidus transactional emails use [Action Mailer][action-mailer], which is built into Rails. Therefore, most
+concepts and customizations that apply to emails in Rails also apply to Solidus. Reviewing
+the [Rails Action Mailer Documentation][action-mailer] will give you some good ideas about how transactional
+emails can be customized in Solidus.
+
+Action Mailer emails have two parts:
+
+1. A view or layout where you can compose and style the email. This could be an `html.erb` file, or a
+   `.txt.erb` file for text-only emails.
+2. A mailer, which behaves very similar to a Rails controller. Like a controller, customizing the mailer will
+   let you determine what information is available in your email's view.
+
+[action-mailer]: https://guides.rubyonrails.org/action_mailer_basics.html
+
+:::
+
+## Customizing your emails
+
+Suppose we want to inform our customers about the estimated time of arrival (ETA) for their packages.
+What we'll need to do is:
+
+1. Customize the right mailer to make it retrieve the ETA for the package.
+2. Customize the email template to render the retrieved ETA along with the other information.
+
+### Override the default mailer
```

Comment:
I think it's a bit confusing to use the word "override" in this example, as we aren't overriding the mailer class in the way that the word override is usually given in Ruby (i.e., re-opening a class/module).

Reply:
Agree, couldn't find a nice name for "Use our own version of the mailer instead of the default one". Do you have any suggestion?

Reply:
What about "replace" instead of "override"? Or "Configure"?